### PR TITLE
openscad: add page

### LINKED ISSUE #6268: page request: openscad
https://en.wikibooks.org/wiki/OpenSCAD_User_Manual/Using_OpenSCAD_in_a_command_line_environment

### PR DESCRIPTION
**Pull request by chrissxYT**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Closes #6268

### CODE REVIEW

File: pages/common/openscad.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# openscad
+
+> Software for creating solid 3D CAD objects.
+> More information: <https://openscad.org>.
+
+- Open a file:
+
+`openscad {{button.scad}}`
```
Comment (einverne) [inline pages/common/openscad.md:8]:
```suggestion
`openscad {{path/to/button.scad}}`
```

File: pages/common/openscad.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Convert a file to STL:
+
+`openscad -o {{button.stl}} {{button.scad}}`
```
Comment (einverne) [inline pages/common/openscad.md:12]:
```suggestion
`openscad -o {{path/to/button.stl}} {{path/to/button.scad}}`
```

File: pages/common/openscad.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Render a file to PNG in a specific colorscheme:
+
+`openscad -o {{button.png}} --colorscheme {{Sunset}} {{button.scad}}`
```
Comment (einverne) [inline pages/common/openscad.md:16]:
```suggestion
`openscad -o {{path/to/button.png}} --colorscheme {{Sunset}} {{path/to/button.scad}}`
```

**Review by einverne — CHANGES_REQUESTED** [review]
Thank you for the new page, but I left a little suggestion for you to take a look. According to the [guideline](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md), when we refer to a file or a directory, we often use the suggestion format.

**Comment by CleanMachine1** [comment]
Closes #6268

File: pages/common/openscad.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# openscad
+
+> Software for creating solid 3D CAD objects.
+> More information: <https://openscad.org>.
+
+- Open a file:
```
Comment (mfrw) [inline pages/common/openscad.md:6]: Can we open only a file with `scad` extension or anything else as well ?

File: pages/common/openscad.md (hunk 2 above)
Comment (CleanMachine1) [reply pages/common/openscad.md:12]: Does this have long flags E.G. -o to --output

File: pages/common/openscad.md (hunk 4 above)
Comment (chrissxYT) [reply pages/common/openscad.md:6]: OpenSCAD only works with `scad` files, something like `openscad x.stl` opens up creating a new `scad` like `import("/path/to/x.stl");`

File: pages/common/openscad.md (hunk 2 above)
Comment (chrissxYT) [reply pages/common/openscad.md:12]: @CleanMachine1 It does not.

**Review by einverne — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
LGTM :)
Thank you for your contribution.

**Pull request merged**